eq-author/src/components/QuestionnaireTable/Row.js

```diff
@@ -12,15 +12,16 @@ import IconButtonDelete from "components/buttons/IconButtonDelete";
 import DuplicateButton from "components/buttons/DuplicateButton";
 import DeleteConfirmDialog from "components/DeleteConfirmDialog";
 import Truncated from "components/Truncated";
+import Button from "components/buttons/Button";
 
 import { buildQuestionnairePath } from "utils/UrlUtils";
 
 import { colors } from "constants/theme";
 import { WRITE } from "constants/questionnaire-permissions";
-import Button from "components/buttons/Button";
+import * as Headings from "constants/table-headings";
 
-import FormattedDate from "./FormattedDate";
-import questionConfirmationIcon from "./icon-questionnaire.svg";
+import FormattedDate from "../../App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/FormattedDate.js";
+import questionConfirmationIcon from "../../App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/icon-questionnaire.svg";
```

Comment:
Can you move this svg into the `assets` directory, please? If there's already one there, you can use that instead and delete the one living in `QuestionnairesTable/row`

See lines 25-27 for an example on how to import an SVG from the assets directory.